src/components/Changelog/IncidentChangelogItems.tsx

```diff
@@ -0,0 +1,28 @@
+import { IncidentHistory } from "../../api/services/IncidentsHistory";
+import { relativeDateTime } from "../../utils/date";
+import { Avatar } from "../Avatar";
+import IncidentHistoryItemTypeContent from "./IncidentHistoryItemTypeContent";
+
+type IncidentChangelogItemProps = {
+  history: IncidentHistory;
+};
+
+export default function IncidentChangelogItem({
+  history
+}: IncidentChangelogItemProps) {
+  return (
+    <li className="flex flex-row items-center space-x-3" key={history.id}>
```

Comment:
- Can we increase the density (i.e. reduce spacing / avatar size) - Maybe use the circle avatar variant
- We should use more font styling/badges to break up the single line of text 
- The evidence addition should be of icon and name e.g. `Added {icon} {name}` without the evidence text